Gemfile.lock

```diff
@@ -1,97 +1,115 @@
     mini_portile2 (2.5.0)
+    minitest (5.14.2)
     mustermann (1.1.1)
       ruby2_keywords (~> 0.0.1)
-    parallel (1.19.1)
-    parser (2.7.0.4)
-      ast (~> 2.4.0)
-    pry (0.12.2)
-      coderay (~> 1.1.0)
-      method_source (~> 0.9.0)
-    pry-byebug (3.8.0)
+    parallel (1.19.2)
+    parser (2.7.1.4)
+      ast (~> 2.4.1)
+    pry (0.13.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    pry-byebug (3.9.0)
       byebug (~> 11.0)
-      pry (~> 0.10)
-    public_suffix (4.0.3)
-    rack (2.2.2)
+      pry (~> 0.13.0)
+    public_suffix (4.0.6)
+    rack (2.2.3)
     rack-protection (2.0.8.1)
       rack
-    rack-test (0.8.3)
+    rack-test (1.1.0)
       rack (>= 1.0, < 3)
     rainbow (3.0.0)
     rake (13.0.1)
+    regexp_parser (1.7.1)
     rexml (3.2.4)
     rspec (3.9.0)
       rspec-core (~> 3.9.0)
       rspec-expectations (~> 3.9.0)
       rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.1)
-      rspec-support (~> 3.9.1)
-    rspec-expectations (3.9.1)
+    rspec-core (3.9.2)
+      rspec-support (~> 3.9.3)
+    rspec-expectations (3.9.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
     rspec-mocks (3.9.1)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
-    rspec-support (3.9.2)
-    rubocop (0.80.1)
-      jaro_winkler (~> 1.5.1)
+    rspec-support (3.9.3)
+    rubocop (0.87.1)
       parallel (~> 1.10)
-      parser (>= 2.7.0.1)
+      parser (>= 2.7.1.1)
       rainbow (>= 2.2.2, < 4.0)
+      regexp_parser (>= 1.7)
       rexml
+      rubocop-ast (>= 0.1.0, < 1.0)
       ruby-progressbar (~> 1.7)
-      unicode-display_width (>= 1.4.0, < 1.7)
-    rubocop-govuk (3.3.0)
-      rubocop (= 0.80.1)
-      rubocop-rails (~> 2)
-      rubocop-rspec (~> 1.28)
-    rubocop-rails (2.4.2)
+      unicode-display_width (>= 1.4.0, < 2.0)
+    rubocop-ast (0.3.0)
+      parser (>= 2.7.1.4)
+    rubocop-govuk (3.17.0)
+      rubocop (= 0.87.1)
+      rubocop-rails (= 2.6.0)
+      rubocop-rake (= 0.5.1)
+      rubocop-rspec (= 1.42.0)
+    rubocop-rails (2.6.0)
+      activesupport (>= 4.2.0)
       rack (>= 1.1)
-      rubocop (>= 0.72.0)
-    rubocop-rspec (1.38.1)
-      rubocop (>= 0.68.1)
+      rubocop (>= 0.82.0)
+    rubocop-rake (0.5.1)
+      rubocop
+    rubocop-rspec (1.42.0)
+      rubocop (>= 0.87.0)
     ruby-progressbar (1.10.1)
     ruby2_keywords (0.0.2)
     sinatra (2.0.8.1)
       mustermann (~> 1.0)
       rack (~> 2.0)
       rack-protection (= 2.0.8.1)
       tilt (~> 2.0)
+    thread_safe (0.3.6)
     tilt (2.0.10)
-    unicode-display_width (1.6.1)
+    tzinfo (1.2.7)
+      thread_safe (~> 0.1)
+    unicode-display_width (1.7.0)
+    zeitwerk (2.4.0)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
-  govuk_schemas (~> 3.2.0)
+  govuk_schemas
   json-schema
-  jsonnet (~> 0.2)
+  jsonnet
   pry-byebug
-  rack-test (~> 0.8.2)
+  rack-test
   rake
   rspec
-  rubocop-govuk (~> 3.3.0)
-  sinatra (~> 2.0)
-
-RUBY VERSION
-   ruby 2.6.6
+  rubocop-govuk
+  sinatra
```

Comment:
when I bundled this locally, these were removed. Don't know what's happening there but might be worth checking out.